docs/.vitepress/components/OldDocument.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <div class="old-document">
+    <p>
+      This documentation covers Vitest 1 <strong>(old version)</strong>. For the
+      latest version, see
+      <a href="https://vitest.dev" class="new-document-link">https://vitest.dev</a>.
+    </p>
+  </div>
+</template>
+
+<style>
+.old-document {
+  position: fixed;
+  top: 0;
+  left: 0;
+  display: flex;
+  height: var(--vp-layout-top-height);
+  width: 100%;
+  padding: 4px 32px;
+  justify-content: center;
+  align-items: center;
+  color: var(--vp-c-text-1);
+  background: var(--vp-c-brand-softest);
```

Comment:
How about making this more visible?

```suggestion
  background-color: var(--vp-c-caution-3);
```

<img src="https://github.com/user-attachments/assets/95895a03-8609-4b11-98d9-ae33b3390c5a" width="400" />